project_management/site_completeness_checklist.md

```diff
@@ -5,16 +5,14 @@ This list uses [Github Checklists](https://github.com/blog/1375-task-lists-in-gf
 
 ## Early Dev Checklist 
 
-* [ ] Dev site disallows indexing
-* [ ] Other item
-* [ ] all states of navigation/buttons (active, hover, inactive, etc)
-* [ ] dropdown menus
-* [ ] pagination
-* [ ] headline styles that work when very long (wrap to multiple lines)

```

Comment:
Are we intending to remove these 4 design items? Not sure these really fall into site completeness anyway.